src/main/java/com/google/sps/data/IDataManager.java

```diff
@@ -27,7 +27,7 @@
     public void updateStage(Stage stage);
     public Stage retrieveStage(String stageID);
 
-    public void storeProgress(Progress progress);
-    public void updateProgress(Progress progress);
-    public Progress retrieveProgress(String progressID);
+    public void storeSingleplayerProgress(SingleplayerProgress progress);
```

Comment:
Why is it SingleplayerProgress not SinglePlayerProgress?

Reply:
I considered "singleplayer" to be one word, but I can change this.